orttraining/orttraining/python/training/ortmodule/_runtime_inspector.py

```diff
@@ -71,7 +66,18 @@ def _initialize_embedding_padding_inspector(self, model, user_input_names):
             if not found or helper.get_attribute_value(found[0]).decode() != "embedding":
                 continue
 
-            tensor = self._try_get_initializer(model, node.input[2])
+            tensor = None
+            padding_const_node = self._try_get_node_from_its_output(node.input[2])
```

Comment:
add check to verify the node has the number of inputs you are expecting. If the node does not have 2 inputs, then this will fail.